packages/playground/src/ErrorBoundary.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+class ErrorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { hasError: false, error: null };
+    this.resetError = this.resetErrorBoundary.bind(this);
+  }
+
+  /** Update state so the next render will show the fallback UI. */
+  static getDerivedStateFromError(error) {
+    return { hasError: true, error: error };
+  }
+
+  resetErrorBoundary() {
+    this.setState({ hasError: false, error: null });
+  }
```

Comment:
If you turn this into a fat-arrow function then you don't need the bind above


```suggestion
  resetErrorBoundary = () => {
    this.setState({ hasError: false, error: null });
  }
```